### ISSUE TITLE
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Program Files\\Google\\Chrome\\Application

### ISSUE THREAD

**Issue by shaiksamad** [yeongbin-jo/python-chromedriver-autoinstaller#71]
## code:

```python

from selenium import webdriver
import chromedriver_autoinstaller


chromedriver_autoinstaller.install()  # Check if the current version of chromedriver exists
                                      # and if it doesn't exist, download it automatically,
                                      # then add chromedriver to path

driver = webdriver.Chrome()
driver.get("http://www.python.org")
assert "Python" in driver.title
driver.quit()

```
## Error:

Traceback (most recent call last):
  File "C:\Users\samad\Desktop\project\main.py", line 5, in <module>
    chromedriver_autoinstaller.install()  # Check if the current version of chromedriver exists
  File "C:\Users\samad\Desktop\project\venv\lib\site-packages\chromedriver_autoinstaller\__init__.py", line 21, in install    
    chromedriver_filepath = utils.download_chromedriver(path, no_ssl)
  File "C:\Users\samad\Desktop\project\venv\lib\site-packages\chromedriver_autoinstaller\utils.py", line 269, in download_chromedriver
    chrome_version = get_chrome_version()
  File "C:\Users\samad\Desktop\project\venv\lib\site-packages\chromedriver_autoinstaller\utils.py", line 166, in get_chrome_version
    dirs = [f.name for f in os.scandir("C:\\Program Files\\Google\\Chrome\\Application") if f.is_dir() and re.match("^[0-9.]+$", f.name)]
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Program Files\\Google\\Chrome\\Application'

**Comment by shaiksamad** [comment]
I can solve this.

**Comment by shaiksamad** [comment]
Fixed in PR #72
